javascript/ql/src/Security/CWE-116/IncompleteHtmlAttributeSanitization.qhelp

```diff
@@ -38,6 +38,14 @@
 
 		</p>
 
+		<p>
+
+			An even safer alternative is to design the application
+			such that sanitization isn't needed at all, for instance by using HTML
```

Comment:
I'd suggest that it's clearer as:

```suggestion
			so that sanitization is not needed, for instance by using HTML
```

Reply:
While the original text adds emphasis to the statement for fluent speakers, it may confuse people who are less proficient.